src/repl-frontend/scala/tools/nsc/interpreter/shell/Reporter.scala

```diff
@@ -154,30 +154,32 @@ class ReplReporterImpl(val config: ShellConfig, val settings: Settings = new Set
       if (colorOk) severityColor(severity) + clabel(severity) + RESET
       else clabel(severity)
 
-    printMessage(pos, prefix + msg)
+    val amended = {
+      val line0 = pos.lineContent
+      var line = line0
+      if (line.nonEmpty && line.indexOf('\n') == -1 && actions.nonEmpty) {
+        val comment = new StringBuilder
+        for (CodeAction(title, description, edits) <- actions; TextEdit(editPos, editText) <- edits)
+          if (pos.line == editPos.line) {
+            line = line.patch(editPos.start, editText, editPos.end - editPos.start)
+            if (comment.nonEmpty) comment.append("; ")
+            comment.append(title)
+          }
+        if (line == line0) ""
+        else if (comment.isEmpty) line
+        else s"$line // $comment" // sb.append(s"$patched // ${description.getOrElse(title)}")
```

Comment:
In a multi-line input only the fixed line is shown, I guess that's alo intended?
```
scala> def f = {
     |   println
     | }
         println
         ^
On line 2: warning: 💡 Auto-application to `()` is deprecated. Supply the empty argument list `()` explicitly to invoke method println,
       or remove the empty argument list from its definition (Java-defined methods are exempt).
       In Scala 3, an unapplied method like this will be eta-expanded into a function.

quick>   println() // add `()`
def f: Unit
```